pkg/sql/sqlbase/system.go

```diff
@@ -104,6 +104,15 @@ CREATE TABLE system.ui (
 	value       BYTES,
 	lastUpdated TIMESTAMP NOT NULL
 );`
+
+	JobsTableSchema = `
+CREATE TABLE system.jobs (
+	id                INT       DEFAULT unique_rowid() PRIMARY KEY,
+	status            STRING    NOT NULL,
+	created           TIMESTAMP NOT NULL DEFAULT now(),
+	payload           BYTES     NOT NULL,
+	INDEX (status, created)
```

Comment:
manually specify the single FAMILY with all columns (in case we ever change the heuristics)